src/ResourceManager/CognitiveServices/CognitiveServices.Test/packages.config

```diff
@@ -15,6 +15,8 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.13" targetFramework="net452" />
```

Comment:
@yangyuan same comment about removing the extra references from this file

Reply:
Removed.